src/initiate.js

```diff
@@ -116,8 +142,31 @@ export const initiateRendering = async (
       outputDir,
       outputFilename,
     );
+
+    console.log(
+      `\x1b[32m${outputFilename}.mbtiles has been successfully generated!\x1b[0m`,
+    );
+
+    // if successful, return the metadata
+    return {
+      style: style,
+      status: metadata.status,
+      errorCode: metadata.errorCode,
+      errorMessage: metadata.errorMessage,
+      filename: metadata.filename,
+      filesize: metadata.filesize,
+      numberOfTiles: metadata.numberOfTiles,
+      numberOfAttempts: 1,
+      workBegun,
+      workEnded: new Date().toISOString(),
```

Comment:
I do think it could be interesting that we return these for the CLI. I say this from first-hand experience of running similar Node tools overnight, and not being sure when it actually finished.